src/ZendeskApi_v2/Requests/Tickets.cs

```diff
@@ -470,6 +473,22 @@ public bool DeleteMultiple(IEnumerable<long> ids)
             return GenericDelete($"{_tickets}/destroy_many.json?ids={ids.ToCsv()}");
         }
 
+        /// <summary>
+        /// Merges the source tickets in the "ids" list into the target ticket with comments as defined.
+        /// </summary>
+        /// <param name="targetTicketId">Id of ticket to be merged into.</param>
+        /// <param name="sourceTicketIds">List of ids of source tickets to be merged from.</param>
+        /// <param name="targetComment">Private comment to add to the target ticket (optional but recommended)</param>
+        /// <param name="sourceComment">Private comment to add to the source ticket(s) (optional but recommended)</param>
+        /// <param name="targetCommentPublic">Whether comment in target ticket is public or private (default = private)</param>
+        /// <param name="sourceCommentPublic">Whether comment in source ticket is public or private (default = private)</param>
+        /// <returns></returns>
+        public JobStatusResponse MergeTickets(long targetTicketId, IEnumerable<long> sourceTicketIds, string targetComment = "", string sourceComment = "", bool targetCommentPublic = false, bool sourceCommentPublic = false)
+        {
+            return GenericPost<JobStatusResponse>($"{_tickets}/{targetTicketId}/merge.json", new { ids = sourceTicketIds.ToArray(), target_comment = targetComment, source_comment = sourceComment, target_comment_is_public = targetCommentPublic, source_comment_is_public = sourceCommentPublic });
```

Comment:
ToArray is not needed.

```suggestion
            return GenericPost<JobStatusResponse>($"{_tickets}/{targetTicketId}/merge.json", new { ids = sourceTicketIds, target_comment = targetComment, source_comment = sourceComment, target_comment_is_public = targetCommentPublic, source_comment_is_public = sourceCommentPublic });
```